packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -207,7 +216,7 @@ contract Bridge is EssentialContract, IBridge {
         // Process message differently based on the target address
         if (
             message.to == address(0) || message.to == address(this)
-                || message.to == address(signalService)
+                || message.to == address(signalService) || addressBanned[message.to]
```

Comment:
Cleaner to just add the other address in `addressBanned` as well I think.

Reply:
I still think we should leave other checks here as-is to avoid operational mistakes, for example, forgetting to ban the signalService. 

Banning other addresses are not so critical but useful.

Reply:
Ah yes good point that it could easily be forgotten if the signal service address is updated or is not yet known when `init` is called.